app/src/main/res/xml/video_audio_settings.xml

```diff
@@ -129,6 +129,13 @@
             app:singleLineTitle="false"
             app:iconSpaceReserved="false" />
 
+        <SwitchPreferenceCompat
+            android:defaultValue="false"
+            android:key="@string/start_main_player_fullscreen_key"
+            android:summary="@string/start_main_player_fullscreen_summary"
+            android:title="@string/start_main_player_fullscreen_title"
+            app:iconSpaceReserved="false" />
```

Comment:
```suggestion
            app:singleLineTitle="false"
            app:iconSpaceReserved="false" />
```

Reply:
Mmmh, isn't there a way to set these attributes in a theme of some sort, so that they do not need to be specified each time?